Cour-de-cassation/label
# Feat/new checklist

## Issue description :
- accept new checklist types
- make checklists clickable to improve review

## Describe your changes :
- modify annotationReportType
- enable clicking on each check

## TODO : 
- update https://github.com/Cour-de-cassation/label/pull/120 in consequence
- intend to understand why defaultExpanded Accodion argument do not work
- ask data team to change 'label' into 'category' : :heavy_check_mark: https://github.com/Cour-de-cassation/label/pull/133/commits/e84228054e529d2654114030c4025556f1fd28f8 

## How to test :
- insert document with new checklists
- check checklist display
- check checklist click mode

## Checklist before requesting a review
